src/MunicipalityRegistry.Api.CrabImport/Infrastructure/Startup.cs

```diff
@@ -81,36 +91,34 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
         {
             StartupHelpers.EnsureSqlStreamStoreSchema<Startup>(streamStore, loggerFactory);
```

Comment:
Also add the following line above this please:
`StartupHelpers.CheckDatabases(healthCheckService, DatabaseTag).GetAwaiter().GetResult();`